corehq/apps/reports/static/reports/javascripts/reports.config.js

```diff
@@ -110,12 +110,12 @@ var HQReport = function (options) {
             $.cookie(self.toggleFiltersCookie, showStr, {path: self.urlRoot, expires: 1});
         };
         
-        if ($.cookie(self.toggleFiltersCookie) === null) {
+        if ($.cookie(self.toggleFiltersCookie) === undefined || $.cookie(self.toggleFiltersCookie) === null) {
```

Comment:
comparing to `undefined` is not considered safe because `undefined` an be reassigned (silly, huh?). The conventional way to do it is `typeof $.cookie(self.toggleFiltersCookie) === "undefined"`. In this case it may also be reasonable to simplify the expression like this:
```js
if (!$.cookie(self.toggleFiltersCookie)) {
```